fetch_data.py

```diff
@@ -2,18 +2,21 @@
 from google.cloud import storage
 from dotenv import load_dotenv
 import os
+from pathlib import Path
 
 
 def configure():
     load_dotenv()
 
-creds = os.getenv("CREDS")
-# Set your GCS credentials
-storage_client = storage.Client.from_service_account_json(creds)
-bucket_name = 'rohan_bkk'
+# Create a storage_client using your GCS credentials
+creds_path = Path(__file__).resolve().parent / 'creds.json'
+creds_path.write_text(os.getenv("CREDS"))
+storage_client = storage.Client.from_service_account_json(str(creds_path))
```

Comment:
This is looking for json text, not for the private key.

Reply:
that service key is in JSON format.

Reply:
sample service_account_key.json :- 
    
    {
          "type": "service_account",
          "project_id": "clean-phoenix-37826532",
          "private_key_id": "6d5c612frgg34t34t4t4tgghh7j8j8787i8kke3e76d15",
          "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCqDm7TfAJYfO5A\nRLXrh5AZjdsm94GaR4+/7yHwpE1FHr1IraZE75/swx42fNIYBdNgpiObVbcsXzEB\n8jzjSBaSlM3mwvO/UAPzD03/POq/eCZuWAkGAuBUZh7LKsOauTjahxAfmnY2wDeo\nQuPIWdCvNGOH9ejBRG9yoQDH3OU5V8D6edY7GnVzXoNOT7KJuIOqANhwDDJyOH7i\nHbvXWR66UBg/FoFobJVSwBjl0oMGIfOAlU8GOz5e5OEaea7rG6yToKe6Gq7AHGFE\nnZ0u4CxRsYkGGBP6uz78YDv+O1KT6d3WlQhJ7kpUay6Ia2WZ/WBVM4kpz+YQwodX\nS0UXLdYLAgMBAAECggEADXg9MQUmjmm+2VpX6oL/V/bTf/+kxVqbuUEwuAXblPj9\nVy9h9x3f1foSf5Bdd8emWYZvVZNxgdM8Z4IPbpU8LHoa4QdaFTI6TdtNe0UQ9v+G\nU8EnXdGGgUjhcYg+nBfFuBFwpoAukuGgxvQdZLYwgGAu7gr9V6mbBkJYdHw7r9Hx\nibtv95jN5DWchQ1K+93mPty+Le7gFumHM53zcecnj3ig+dQLzDSNiKiG1TAfZQPi\nISGVqTgnoX71fz11cKILicoMZqwaaubYHGmI+q4xwSHN0k1CpXhsGoO9y1aizm5z\np2UArw1yBW3b9ccATA0p0eXBvzqfanGWxQ1HtUElSQKBgQDTYcPQ5PKsF+3DPXcG\nTW6UclBbZ7zHLXwibtxlC13GsrKHvNqoZhh4MJhaTa5GxV/E9a2RmGocdiTbNB4x\nX3N8u11u48p1NDGN8g3NCmXHEiX/677OEJ2ESlBslY0+rSchohs3V0HFqEEavn3W\nKYdtDpQ79GCFLoAacOYexY2e7wKBgQDN85oSzFP/wJAg8HR17jol0n5IVFpdt5AF\nM+0BY3ONwOMHYyndoeX0VqyHUzMmQE0Mg4WccjcThq44Y8U9bhe7RQZn5gU15ZxD\npujocdi3Pq3co6QJ9Zp4hDSVtIHI42T7I9LtNA+KmL1diMiFLc/cJMNR5Pt0KcSC\nw8XSNV/6pQKBgQC543LRZFVoM87acFYYi8Wf8/totoyT2BMfMMDOdzVrhEBuEA12\nI/qGz61bVI7bG72Bts+BhzhheSAmSOn4X0lvOlILL3Ueptyf5qZ0tDGQOtZbtmEO\naRAXGNeGJTeo2QsXLZly6QzcCScGznRSykuzO2xGs5bgjxzxZQNwEjCz+S/69WCA\nKmtyySttJZEScs59wDR9zPbt\n-----END PRIVATE KEY-----\n",
          "client_email": "me-948@clean-phoenix-378215.iam.gserviceaccount.com",
          "client_id": "1023123094534343453454535",
          "auth_uri": "https://accounts.google.com/o/oauth2/auth",
          "token_uri": "https://oauth2.googleapis.com/token",
          "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
          "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/me-948%40clean-phoenix-378215.iam.gserviceaccount.com"
        }